src/osm_configurator/model/project/calculation/aggregation_method_enum.py

```diff
@@ -72,3 +72,9 @@ def get_name(self):
             str: Name of the enum type
         """
         return self.value[1]
+
+    def equals(mode: str) -> AggregationMethod | None:
```

Comment:
better name or descirbe in pydoc what this method does.
A "normal" equals method would return a boolean.